src/nodes/Navigation/NavigationInfo.js

```diff
@@ -62,6 +62,14 @@ x3dom.registerNodeType(
 
             /**
              * Specifies the view angle and height for helicopter mode and min/max rotation angle for turntable in ]0, PI[, starting from +y (0) down to -y (PI)
+             * [0] helicopter vertical view angle theta
+             * [1] helicopter height
+             * [2] min vertical view angle theta

```

Comment:
turntable theta max

Reply:
I guess `typeParams[ 2 ]` is the min value and `typeParams[ 3 ]` is the max value:

`NavigationInfo.js`
```
getTypeParams : function ()
            {
                var length = this._vf.typeParams.length;

                var theta  = ( length >= 1 ) ? this._vf.typeParams[ 0 ] : -0.4;
                var height = ( length >= 2 ) ? this._vf.typeParams[ 1 ] : 60.0;
                var minAngle = ( length >= 3 ) ? this._vf.typeParams[ 2 ] : x3dom.fields.Eps;
                var maxAngle = ( length >= 4 ) ? this._vf.typeParams[ 3 ] : Math.PI - x3dom.fields.Eps;
```

This match to the usage here:
`TurntableNavigation.js`
```
    // clamp theta
    theta = Math.max( navi._vf.typeParams[ 2 ], Math.min( navi._vf.typeParams[ 3 ], theta ) );
```

What do you think about:
```
             * [2] turntable min vertical view angle theta
             * [3] turntable max vertical view angle theta
```